docs/calicoctl/profile.md

```diff
@@ -0,0 +1,518 @@
+
+    <PROFILE>: Name of profile to remove.
+
+    --no-check: Remove the container without checking endpoint dependencies.
+```
+
+Examples:
+
+```
+$ calicoctl profile remove PROF_TEST
+Deleted profile PROF_TEST
+```
+
+### calicoctl profile <PROFILE> tag show
+This command shows the tags related to a given profile.
+
+Tags are used to configure policy to allow/deny traffic 
+from endpoints associated with profiles containing the 
+given tags.  See the `calicoctl profile <PROFILE> rule add` 
+command below for additional information.
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl profile <PROFILE> tag show
+
+    <PROFILE>: Name of profile whose tags will be printed.
+```
+
+Examples:
+
+```
+$ calicoctl profile PROF_TEST tag show
+PROF_TEST
+```
+
+### calicoctl profile <PROFILE> tag (add|remove) <TAG>
+This command allows you to add or remove a profile tag from the given profile. 
+
+An example use case for adding tags would be if you have multiple profiles 
+that serve a similar purpose, such as a group of web applications. You may have 
+backend service that talks to all of these web applications. This backend 
+service is using a profile called BACKEND.  It would be tedious to create 
+multiple rules on the BACKEND profile for each web application that the backend 
+service interacts with.  Instead, you can set a tag on the web application 
+profiles called WEB_APP, then create a rule on the BACKEND profile that allows 
+interaction from the WEB_APP tag.
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl profile <PROFILE> tag (add|remove) <TAG>
+
+    <PROFILE>: Name of profile to add or remove a tag from.
+    <TAG>: Tag name to add or remove from profile.
+```
+
+Examples:
+
+```
+# Add the WEB_APP tag to the WEB_SERVER_A profile
+$ calicoctl profile WEB_SERVER_A tag add WEB_APP
+Tag WEB_APP added to profile WEB_SERVER_A
+
+# View the tags of the WEB_SERVER_A profile
+$ calicoctl profile PROF tag show
+WEB_SERVER_A
+WEB_APP
+
+# Remove the WEB_APP tag from the WEB_SERVER_A profile
+$ calicoctl profile WEB_SERVER_A tag remove WEB_APP
+Tag WEB_APP removed from profile WEB_SERVER_A
+```
+
+### calicoctl profile <PROFILE> rule add (inbound|outbound) (allow|deny)
+This command allows you to configure policy rules on your 
+security profiles.
+
+You can configure rules to allow and/or deny specific traffic 
+to and from your containers, based on a variety of criteria.
+
+You can filter traffic based on any combination of the following:
+ - *Type* - udp, tcp, icmp (including icmp type and code)
+ - *Source tag* - Profile tag, such as WEB_SERVER
+ - *Source cidr* - such as 172.25.1.0/24
+ - *Source port* - TCP/UDP only
+ - *Destination tag* - Profile tag, such as DATA_SERVER
+ - *Destination cidr* - such as 172.25.2.0/24
+ - *Destination port* - TCP/UDP only
+
+#### Docker default networking
+When a profile is created with Docker default networking, the 
```

Comment:
Might be easier to say "When a profile is created with `calicoctl profile add`..."